sympy/parsing/latex/_parse_latex_antlr.py

```diff
@@ -82,6 +83,8 @@ def parse_latex(sympy):
     parser.addErrorListener(matherror)
 
     relation = parser.math().relation()
+    if strict and (relation.start.start != 0 or relation.stop.stop != len(sympy) - 1):
+        raise LaTeXParsingError("Invalid LaTeX")
```

Comment:
Is there any equivalent antlr option?

Reply:
I think ANTLR does not provide such an option. It needs to be implemented during parsing and in the grammar/lexical definitions.